examples/vnet/gw_port.c

```diff
@@ -44,10 +44,15 @@ static void gw_assert_link_status(int port_id)
 		rte_exit(EXIT_FAILURE, ":: error: link is still down\n");
 }
 
-int gw_init_port(int port_id, int nr_queues)
+int sf_start_dpdk_port(struct sf_port_cfg *port)
```

Comment:
not following why you changed the function name

Reply:
because we start the port here we cann "rte_eth_dev_start".
after starting the port we need to bind the queues to the ports. can be reverted if you want